nemo_skills/dataset/aalcr/prepare.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) 2024, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import argparse
+import json
+from pathlib import Path
+
+import tiktoken
+from datasets import load_dataset
+from tqdm import tqdm
+
+"""
+Usage
+# default. setup is aalcr (all).
+python prepare.py
+
+# prepare subset aalcr_100k.
+python prepare.py --max_context_window 100000 --setup aalcr_100k
+
+or
+ns prepare_data \
+    --data_dir=/workspace/ns-data \
+    --cluster=fei-ord \
+    aalcr --txt_file_folder=/workspace/do_not_share_data/lcr
+"""
+
+
+def construct_prompt(docs, question):
+    documents_text = "\n\n".join(f"BEGIN DOCUMENT {i + 1}:\n{doc}\nEND DOCUMENT {i + 1}" for i, doc in enumerate(docs))
+    prompt = """BEGIN INPUT DOCUMENTS
+
+    {documents_text}
```

Comment:
this is going to have a bunch of spaces, is this intentional? If not you should remove indentation or use a different string formatting

Reply:
why don't we have it as a normal yaml prompt similar to other benchmarks?